include/parse_arguments.hpp

```diff
@@ -59,10 +59,17 @@ struct Arguments {
   bool plot_polygons;
   bool plot_quadtree;
   bool skip_projection;
+
+  // Column names in provided visual variables file (CSV)
+  std::optional<std::string> id_col;
+  std::optional<std::string> area_col;
```

Comment:
Good idea to use `optional` here. It correctly represents the concept we're trying to tackle.